src/irep2/irep2_expr.h

```diff
@@ -499,6 +499,29 @@ class arith_2ops : public arith_ops
     const expr2tc &v2)
     : arith_ops(t, id), side_1(v1), side_2(v2)
   {
+#ifndef NDEBUG
```

Comment:
The same here. I would recommend adding some comments.

Reply:
I would keep this in the `.h`, otherwise it'll slow down Release-builds unnecessarily.

Reply:
I've added a comment.